ovos_utils/file_utils.py

```
@@ -390,40 +390,50 @@ def shutdown(self):
+
+        def on_any_event(self, event):
+            if event.is_directory:
+                return
+            with self._lock:
+                if event.event_type == "closed":
+                    if event.src_path in self._changed_files:
+                        self._changed_files.remove(event.src_path)
+                        # fire event, it is now safe
+                        try:
+                            self._callback(event.src_path)
+                        except:
```

Ruff (0.14.0)

Do not use bare `except`

(E722)